src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx

```diff
@@ -155,14 +162,64 @@ describe('Step Users', () => {
     await goToUsersStep();
     await clickAddUser();
     await addSshKey('ssh');
+    await addPassword('inval');
     await addUserName('bestUser');
     const invalidUserMessage = screen.getByText(/invalid ssh key/i);
     await waitFor(() => expect(invalidUserMessage));
   });
+
+  test('try to create Azure image with invalid password', async () => {
+    const user = userEvent.setup();
+    await renderCreateMode();
+    await goToRegistrationStep();
+    await clickRegisterLater();
+    await goToUsersStep();
+    await clickAddUser();
+    const imageOutput = screen.getByRole('button', {
+      name: /image output/i,
+    });
+    await user.click(imageOutput);
+    const azureTarget = screen.getByText(/microsoft azure/i);
+    await user.click(azureTarget);
+    const users = screen.getByRole('button', {
+      name: /users/i,
+    });
+    await user.click(users);
+    await addUserName(validUserName);
+    await addPassword('inval');
+
+    const invalidUserMessage = screen.getByText(
+      /a password for the target environment azure must be at least 6 characters long\. please enter a longer password\./i
+    );
+    const warningUserMessage = screen.getByText(
+      /warning: this password seems weak, please use with caution or include at least 3 of the following: lowercase letter, uppercase letters, numbers, symbols/i
+    );
+    await waitFor(() => expect(invalidUserMessage));
+    await waitFor(() => expect(warningUserMessage));
+    const nextButton = await getNextButton();
+    await waitFor(() => expect(nextButton).toBeDisabled());
+  });
+
+  test('with invalid password', async () => {
+    await renderCreateMode();
+    await goToRegistrationStep();
+    await clickRegisterLater();
+    await goToUsersStep();
+    await clickAddUser();
+    await addUserName(validUserName);
+    await addPassword('inval');
+
+    const invalidUserMessage = screen.getByText(
+      /Password helps protect your account, we recommend a password of at least 6 characters./i
+    );
+    await waitFor(() => expect(invalidUserMessage));
+    const nextButton = await getNextButton();
+    await waitFor(() => expect(nextButton).toBeDisabled());
+  });
 });
 
 describe('User request generated correctly', () => {
-  test('with valid name, ssh key and checked Administrator checkbox', async () => {
+  test('create Guest-Image with valid name, password, ssh key and checked Administrator checkbox', async () => {
```

Comment:
This is very nitpicky - do we have to specify the guest-image here? I mean this should work for all targets, but Azure. So I would just mention Azure where it's applicable as it makes more sense to stress an exception than specify the "default".